models/src/main/thrift/content/v1.thrift

```diff
@@ -1206,6 +1206,8 @@ struct Package {
 
     /* The package name */
     3: required string packageName
+
+    4: optional CapiDateTime lastModified
```

Comment:
@tomrf1 I think this field is required, and you put it optional for compatibility?
In that case, @cb372 told me this is unnecessary.